lib/model/recent_senders.dart

```diff
@@ -68,6 +68,52 @@ class RecentSenders {
       [senderId] ??= MessageIdTracker()).add(messageId);
   }
 
+  /// Handles channel/topic updates to the data resulted from a move.
```

Comment:
I having trouble parsing this sentence. Perhaps: "Handles channel/topic updates when messages are moved."?